apps/server/src/services/project-service/ProjectService.ts

```diff
@@ -209,7 +210,7 @@ export async function applyDataModel(data: Partial<DatabaseModel>, _options?: Op
   // TODO: allow partial project merge from options
   const { rundown, customFields, ...rest } = data;
   const newData = await DataProvider.mergeIntoData(rest);
-
+  startIntegrations();
```

Comment:
hmm, we cant really do this... or shouldnt at least.

inner service layers should not have dependencies on the outer layers

Reply:
Hmm
So we should trigger it from the project controller?

Or?

Reply:
I think that would work. At least to maintain the flow inwards into the app

I think we already had similar problems that we solved? Like with restarting some services when the options change?

It sounds to me that when we apply a new project there will be several things to reset, so it could make sense to expose a pattern to reset the services